server/lib/util.js

```diff
@@ -243,6 +244,14 @@ function createDirectory(dirPath) {
   }
 }
 
+
+function getRandomColor() {
+  const r = Math.floor(Math.random() * 255);
+  const g = Math.floor(Math.random() * 255);
+  const b = Math.floor(Math.random() * 255);
+  return `rgb(${r},${g},${b})`;
```

Comment:
You may consider looking into CSS's [`hcl()`](https://www.w3schools.com/csSref/func_hsl.asp) for better color consistency.

Reply:
[Another resource](http://hclwizard.org/color-scheme/) explaining the science behind it.